src/eval/func.rs

```diff
@@ -144,6 +145,31 @@ impl Func {
             _ => None,
         }
     }
+
+    /// Get a field from this function's scope,
+    /// if possible.
+    pub fn get(&self, field: &str) -> StrResult<&Value> {
+        match &self.repr {
+            Repr::Native(func) => func.info.scope.get(field).ok_or_else(|| {
+                eco_format!(
+                    "function `{}` does not contain field `{}`",
+                    func.info.name,
+                    field
+                )
+            }),
+            Repr::Elem(func) => func.info().scope.get(field).ok_or_else(|| {
+                eco_format!(
+                    "function `{}` does not contain field `{}`",
+                    func.name(),
+                    field
+                )
+            }),
+            Repr::Closure(_) => Err(eco_format!(
+                "cannot access fields on closures and user-defined functions"
```

Comment:
"closure" isn't a term that is exposed to the user. 
```suggestion
                "cannot access fields on user-defined functions"
```